build.gradle

```diff
@@ -758,6 +763,7 @@ project(':connect:api') {
   }
 
   javadoc {
+    include "**/org/apache/kafka/connect/**" // needed for the `javadocAll` task
```

Comment:
@ewencp, since we didn't have inclusions or exclusions before and given the project name, I assume that all the classes are public API. Is that correct?